app/deck/gateway/ping.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Check decK connectivity
+---
+
+The `deck file ping`command checks that `deck` can contact the Admin API. This could be either the Konnect API, or an on-prem installation.
```

Comment:
```suggestion
The `deck file ping` command checks that decK can contact the Admin API. This could be either the {{site.konnect_short_name}} API, or an on-prem installation.
```